### ISSUE TITLE
JAWS does not announce placeholder text in input field

### ISSUE THREAD

**Issue by kabsas** [FreedomScientific/standards-support#745]
### Summary

JAWS does not announce placeholder text in input. This is an issue with the virtual cursor on and off.

Steps to reproduce:

1. Launch JAWS
2. Go to https://www.w3schools.com/tags/tryit.asp?filename=tryhtml5_input_placeholder
3. Navigate to phone number input field

### Expected result

JAWS announces the field label followed by the placeholder text

### Actual result

JAWS announces the field label but not the placeholder text

### Additional Information

JAWS version and build number
2023.2307.37

Operating System and version
Windows 10 Enterprise 19045.3208

Browser and version:
Chrome 115
Edge 115
Firefox 116
